docs/can-canjs/canjs.md

```diff
@@ -50,7 +50,7 @@ by extending [can-component] with:
   my-counter button {margin-left: 15px;}
 </style>
 <script type="text/steal-module">
-var Component = require("can-component");
+const Component = require("can-component");
```

Comment:
I think this should be an import statement, but babel wasn’t loading for me when I tested the site locally, so I didn’t want to mess with it.